test/patch.jl

```diff
@@ -56,7 +56,7 @@ end
 
 
 function _aws_http_request_patch(response::HTTP.Messages.Response=_response())
-    return @patch AWS._http_request(request::Request) = response
+    return @patch AWS._http_request(::AWS.HTTPBackend, request::Request) = response
```

Comment:
Patch should work on any backend
```suggestion
    return @patch AWS._http_request(::AWS.AbstractBackend, request::Request) = response
```